docs/notes/extensions.md

```diff
@@ -25,7 +25,12 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 
 ## Integrations
 
-### Angular
+### Angular2+
+
+ - <a href="https://github.com/emn178/angular2-chartjs" target="_blank">emn178/angular2-chartjs</a>
+ - <a href="https://github.com/valor-software/ng2-charts" target="_blank">valor-software/ng2-charts</a>
+
+### AngularJS (v1)
```

Comment:
For consistency, can we use the same format for both titles: `### Angular (v2+)` and `### Angular (v1)`